src/addons/Portal/Portal.js

```diff
@@ -146,6 +147,33 @@ function Portal(props) {
   // Component Event Handlers
   // ----------------------------------------
 
+  React.useEffect(() => {
+    if (!hideOnScroll) {
+      return
+    }
+    const abortController = new AbortController()
+
+    window.addEventListener(
+      'scroll',
+      (e) => {
+        debug('handleHideOnScroll()')
+
+        // Do not hide the popup when scroll comes from inside the popup
+        // https://github.com/Semantic-Org/Semantic-UI-React/issues/4305
+        if (_.isElement(e.target) && contentRef.current.contains(e.target)) {
+          return
+        }
+
+        closePortal(e)
+      },
+      { signal: abortController.signal, passive: true },
```

Comment:
`signal` is nice, but does not really work with old browsers. SUIR & SUI still work in IE11, so let's break people another time :)

For now, let's change it to `.removeEventListener()` 🐱

Reply:
Re `passive`: that's fine. It will be just ignored.